stable/openvpn/Chart.yaml

```diff
@@ -1,22 +1,23 @@
 apiVersion: v1
-description: A Helm chart to install an openvpn server inside a kubernetes cluster.  Certificate generation is also part of the deployment, and this chart will generate client keys as needed.
+description: A Helm chart to install an openvpn server inside a kubernetes cluster.  Certificate
+  generation is also part of the deployment, and this chart will generate client keys
+  as needed.
 name: openvpn
-version: 1.1.2
+version: 2.0.0
```

Comment:
Please update only the minor version

Reply:
IMO this is a breaking change to the chart's API. 

[A previous discussion on a similar PR for stable/acs-engine-autoscaler](https://github.com/kubernetes/charts/pull/1976#discussion_r136875811)

If this is a breaking API change, then I think we should be bumping the major version.